app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,137 @@
+    <string name="Charging">Oplaadanimatie</string>
+
+    <string name="chargingState_error">De huidige oplaadstatus kan niet worden getoond</string>
+    <string name="battery">\??%</string>
+    <string name="clock">H:mm</string>
+    <string name="clock2">HH\nMM</string>
+    <string name="date">d MMM EEE</string>
+    <string name="date2">d MMM EEEE</string>
+    <string name="feature">-</string>
+    <string name="percent">%1$d%%</string>
+    <string name="charged">%1$d%% opgeladen</string>
+
+    <string name="shortcut_settings">Instellingen</string>
+    <string name="shortcut_disabled">Uitgeschakeld</string>
+
+    <!-- Strings related to Settings -->
+    <string name="pref_int_failed">Deze waarde kan niet worden gebruikt</string>
+
+    <string name="pref">Instellingen</string>
+    <string name="pref_general">Algemeen</string>
+    <string name="pref_enabled">Ingeschakeld</string>
+    <string name="pref_disabled">Uitgeschakeld</string>
+    <string name="pref_always_on">Altijdaan-scherm</string>
+    <string name="pref_headphones">Koptelefoonanimatie</string>
+    <string name="pref_charging">Oplaadanimatie</string>
+
+    <string name="pref_categories">Categorieën</string>
+    <string name="pref_look_and_feel">Uiterlijk</string>
+    <string name="pref_look_and_feel_summary">Pas het uiterlijk van het altijdaan-scherm aan</string>
+    <string name="pref_look_and_feel_light">Licht thema</string>
+    <string name="pref_look_and_feel_light_summary">Gebruik het lichte thema</string>
+    <string name="pref_look_and_feel_hour">12-uursklok gebruiken</string>
+    <string name="pref_look_and_feel_hour_summary_on">Huidige instelling: 12-uursklok</string>
+    <string name="pref_look_and_feel_hour_summary_off">Huidige instelling: 24-uursklok</string>
+    <string name="pref_look_and_feel_am_pm">AM/PM tonen</string>
+    <string name="pref_look_and_feel_am_pm_summary">Toont AM/PM op de 12-uursklok</string>
+
+    <string name="pref_ao_settings">Scherminstellingen</string>
+    <string name="pref_ao_settings_summary">Meer instellingen omtrent het altijdaan-scherm</string>
+    <string name="pref_look_and_feel_ao">Uiterlijk</string>
+    <string name="pref_look_and_feel_ao_summary">Pas het uiterlijk van het altijdaan-scherm aan</string>
+    <string name="pref_look_and_feel_ao_array_display_0">Google-achtige stijl</string>
+    <string name="pref_look_and_feel_ao_array_display_1">Samsung-achtige stijl</string>
+    <string name="pref_look_and_feel_ao_array_display_2">Alternatieve Samsung-achtige stijl</string>
+    <string name="pref_ao_watch_face">Uurwerk</string>
+    <string name="pref_ao_showClock">Tijd tonen</string>
+    <string name="pref_ao_showDate">Datum tonen</string>
+    <string name="pref_ao_showBatteryIcn">Accupictogram tonen</string>
+    <string name="pref_ao_showBattery">Accupercentage tonen</string>
+    <string name="pref_ao_showNotifications">Aantal meldingen tonen</string>
```

Comment:
It would be great if you could add the string
```
    <string name="pref_ao_showNotificationIcons">Display notification icons</string>
```
as well!

Reply:
Done! :)